### PR TITLE
PR #1

### PR DESCRIPTION
**Pull request by Alexandr-Romanenko**

### CODE REVIEW

**Comment by Alexandr-Romanenko** [comment]
Hi, i made some improvment to your code

File: str (2).py [hunk 1]
```diff
@@ -24,7 +30,7 @@
 # через два слеша
 ##os.walk('D:\\')
 
-
```
Comment (Alexandr-Romanenko) [inline str (2).py:27]: # com 1

File: str (M) index.py [hunk 2]
```diff
@@ -4,11 +4,19 @@
...
 # будет искать 'е' между 5 и 10 индексами
+
+try to this way
+for i in str1:
+  if i == 'e':
+    print (i)
+    
 r2 = str1.index('e', 5, 10)
 r3 = str1.index('my w', 0, -1)
-print(r2)
```
Comment (Alexandr-Romanenko) [inline str (M) index.py:9]: com 2

File: 123.py [hunk 3]
```diff
@@ -4,6 +4,8 @@
 int1 = int (input ('Type an integer 1: '))
 int2 = int (input ('Type an integer 2: '))
 
+l = (lambda x, y: True sum(x,y) == 5 or  (x-y) == 5 else False) (int1, int2)
```
Comment (AstonMartinLim) [inline 123.py:7]: This code dosn`t work

Comment (Alexandr-Romanenko) [reply 123.py:7]: I changed code

**Pull request merged**